docs/conf.py

```diff
@@ -9,6 +9,39 @@
 
 from rocm_docs import ROCmDocs
 
+linkcheck_timeout = 10
```

Comment:
should move all this to rocm-docs-core

Reply:
https://github.com/RadeonOpenCompute/rocm-docs-core/pull/117 

will update here once this PR is merged

Reply:
Done in [update linkcheck configs](https://github.com/RadeonOpenCompute/ROCm/pull/2078/commits/8158e89975c3b8d92060d120535fc071da74550d)

General linkcheck configurations were moved to rocm-docs-core. The links to ignore specific to ROCm remain.